fix naming to match litmus-2.1.3 helm chart in ingress doc

fc92 opened the pull request:
deployment names do no seem up to date, at least compared to what is deployed with the litmus-2.1.3 helm chart

<!--  Thanks for sending a pull request!  Here are some tips for you -->

**What this PR does / why we need it**:
Update the doc for ingress deployment to match recent helm chart deployment names and update ingress api version

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
I didn't check consistency with other deployment methods.

**Checklist:**
-   [x] Fixes #137 
-   [x] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+
+| Type      | Description              | Tested K8s Platform                                               |
+| ----------| ------------------------ | ------------------------------------------------------------------|
+| CoreDNS   | CoreDNS pod delete experiment | KubeADM, Minikube  |
```
```suggestion
| CoreDNS   | CoreDNS pod delete experiment | Kubeadm, Minikube  |
```

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+## Prerequisites
+
+- Ensure that the Litmus Chaos Operator is running
+- Ensure that the `coredns-pod-delete` experiment resource is available in the cluster. If not, install from [here](https://hub.litmuschaos.io/api/chaos?file=charts/coredns/coredns-pod-delete/experiment.yaml)
```
lets add command to check (refer: drain node)

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+## Entry Criteria
+
+- CoreDNS pods are healthy before chaos injection
+- Service resolution checks by deploying a dummy nginx application
```
```suggestion
- Service resolution works successfully as determined by deploying a sample nginx application
```

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+## Exit Criteria
+
+- CoreDNS pods are healthy post chaos injection
+- Service resolution checks by deploying a dummy nginx application
```
we can add above line and also about the liveness app

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+
+- Causes graceful pod failure of an coreDNS application
+- Tests deployment sanity (replica availability & uninterrupted service) and recovery workflow of the service
+- Deletion of CoreDNS pod will failed the kubernetes service resolution.
```
do we need this? if the CoreDNS deployment is not resilient (say, not enough replicas), then service resolution will be failed.

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+
+### Prepare ChaosEngine
+
+- Provide the application info in `spec.appinfo`
```
comment - about coreDNS ns (kube-system), labels

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
```
engine-coredns

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,118 @@
+
+- View coredns pod terminations & recovery by setting up a watch on the coredns pods in the application namespace
+
+  `watch -n 1 kubectl get pods -n <chaos-namespace>`
```
may also want to check coreDNS pods health in kube-system

ksatchit commented on docs/disk-loss.md:
```diff
@@ -29,7 +29,7 @@ sidebar_label: Disk Loss
 kind: Secret
 metadata:
-  name: cloud_secret
+  name: cloud-secret
```
:+1:

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,167 @@
+
+- Causes graceful pod failure of an coreDNS replicas
+- Tests deployment sanity (replica availability & uninterrupted service) and recovery workflow of the service
+- Service resolution will failed if coredns replicas will not present.
```
Will fail if coredns replicas are not present

ksatchit commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,167 @@
+
+- This Chaos Experiment can be triggered by creating a ChaosEngine resource on the cluster. To understand the values to provide in a ChaosEngine specification, refer [Getting Started](getstarted.md/#prepare-chaosengine)
+
+- Follow the steps in the sections below to prepare the ChaosEngine & execute the experiment.
```
There is a slightly different line we are using here - pls refer other experiments

chandankumar4 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,167 @@
+| ----------------------|-----------------------------------------------------|-----------|-----------------|
+| TOTAL_CHAOS_DURATION  | The time duration for chaos insertion (seconds)     | Mandatory | Defaults to 15s |
+| CHAOS_INTERVAL        | Time interval b/w two successive pod failures (sec) | Mandatory | Defaults to 5s | LIB                   | The chaos lib used to inject the chaos                | Optional  | Defaults to `litmus`, Supported: `litmus`  |
+
```
can we have the list of all supported variables here like what you have mentioned in the example of chaosengine yaml

amitbhatt818 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,167 @@
+| CoreDNS   | CoreDNS pod delete experiment | Kubeadm, Minikube      |
+
+## Prerequisites
+- Ensure that Litmus is install. If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/)
```
```Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://github.com/litmuschaos/chaos-operator/blob/master/deploy/operator.yaml)```
Use this line @rajdas98

amitbhatt818 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,167 @@
+  chaosType: 'infra'    # It can be infra only
+  components:
+    runner:
+       image: litmuschaos/ansible-runner:ci
```
```components:
    runner:
      image: "litmuschaos/chaos-executor:1.0.0"
      type: "go"``` 
Use this here for go executer

amitbhatt818 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,167 @@
+    appns: kube-system
+    applabel: 'k8s-app=kube-dns'
+    appkind: deployment
+  chaosType: 'infra'    # It can be infra only
```
comment should come above the line

chandankumar4 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,175 @@
+| CoreDNS   | CoreDNS pod delete experiment | Kubeadm, Minikube      |
+
+## Prerequisites
+- Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://github.com/litmuschaos/chaos-operator/blob/master/deploy/operator.yaml)
```
Update the operator link with latest from litmuschaos/pages

chandankumar4 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,175 @@
+| APP_NAMESPACE         | Namespace in which application pods are deployed	  | Mandatory | Defaults to kube-system |
+| APP_LABEL             | Unique Labels in `key=value` format of application deployment	 | Mandatory | Defaults to k8s-app=kube-dns |
+| APP_KIND              | This is describe the application Kind | Mandatory | Mandatory | Defaults to Deployment | 
+| CHAOS_NAMESPACE       | This is a chaos namespace which will create all infra chaos resources in that namespace	| Mandatory | Default to kube-system |
```
```suggestion
| CHAOS_NAMESPACE       | This is a chaos namespace which will create all infra chaos resources in that namespace	| Mandatory | Default to `kube-system` |
```

chandankumar4 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,175 @@
+| CHAOS_INTERVAL        | Time interval b/w two successive pod failures (sec) | Mandatory | Defaults to 5s  | 
+| APP_NAMESPACE         | Namespace in which application pods are deployed	  | Mandatory | Defaults to kube-system |
+| APP_LABEL             | Unique Labels in `key=value` format of application deployment	 | Mandatory | Defaults to k8s-app=kube-dns |
+| APP_KIND              | This is describe the application Kind | Mandatory | Mandatory | Defaults to Deployment | 
```
```suggestion
| APP_KIND              | This is describe the application Kind | Mandatory | Mandatory | Defaults to `Deployment` | 
```

chandankumar4 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,175 @@
+| TOTAL_CHAOS_DURATION  | The time duration for chaos insertion (seconds)     | Mandatory | Defaults to 15s |
+| CHAOS_INTERVAL        | Time interval b/w two successive pod failures (sec) | Mandatory | Defaults to 5s  | 
+| APP_NAMESPACE         | Namespace in which application pods are deployed	  | Mandatory | Defaults to kube-system |
+| APP_LABEL             | Unique Labels in `key=value` format of application deployment	 | Mandatory | Defaults to k8s-app=kube-dns |
```
```suggestion
| APP_LABEL             | Unique Labels in `key=value` format of application deployment	 | Mandatory | Defaults to `k8s-app=kube-dns` |
```

chandankumar4 commented on docs/coredns-pod-delete.md:
```diff
@@ -0,0 +1,175 @@
+| ----------------------|-----------------------------------------------------|-----------|-----------------|
+| TOTAL_CHAOS_DURATION  | The time duration for chaos insertion (seconds)     | Mandatory | Defaults to 15s |
+| CHAOS_INTERVAL        | Time interval b/w two successive pod failures (sec) | Mandatory | Defaults to 5s  | 
+| APP_NAMESPACE         | Namespace in which application pods are deployed	  | Mandatory | Defaults to kube-system |
```
```suggestion
| APP_NAMESPACE         | Namespace in which application pods are deployed	  | Mandatory | Defaults to `kube-system` |
```

fc92 committed:
names matching litmus-2.1.3 helm chart

deployment names do no seem up to date, at least compared to what is deployed with the litmus-2.1.3 helm chart

fc92 committed:
Sign off

Signed-off-by: Florent CARRE <florent.carre@free.fr>

fc92 committed:
Sign off

Signed-off-by: Florent CARRE <florent.carre@free.fr>

fc92 committed:
Merge branch 'patch-1' of github.com:sietchcode/litmus-docs into HEAD

Signed-off-by: Florent CARRE <florent.carre@free.fr>
